external/glfw/lib/android/android_window.c

```diff
@@ -178,6 +178,8 @@ int _glfwPlatformOpenWindow( int width__, int height__,
         update_width_height_info(&_glfwWin, &_glfwWinAndroid, 1);
         computeIconifiedState();
     }
+    
+    _glfwTerminateJoysticks();
```

Comment:
It's important to clean up the new static structure between restarts to make sure `DeviceId` isn't 0.

Reply:
It's a good question, I'm not entirely sure.
Maybe this one https://github.com/defold/defold/pull/10541
or maybe this one https://github.com/defold/defold/pull/10229
not sure